src/Auth/UnattendedAuthentication.cs

```diff
@@ -0,0 +1,61 @@
+﻿using IdentityModel.Client;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Jpp.Common.Backend.Auth
+{
+    public class UnattendedAuthentication : BaseOAuthAuthentication
```

Comment:
Consider sealed, so we explicitly restrict the inheritance.

Reply:
I dont mind it being inherited, personally I dont like sealed by defaulty unless there is a reason for a class to prevent inheritence.

Reply:
Given the nature of the class (warnings, etc in comments), I was considering seal to explicitly prevent inheritance and reduce the risk.